### PR TITLE
Replace confusing 'version' terms with 'origin' terms

### LINKED ISSUE #1183: Package "versions" terminology is confusing
Right now we're using the term "version" to refer to the app version (i.e. version 12.3.4) as well as duplicate app IDs from different sources (i.e. GNOME Builder from Flathub and GNOME Builder from the Ubuntu repos).

The former makes sense to me, but I think we should rename the latter to "sources," "remotes," or something less confusing.

### PR DESCRIPTION
**Pull request by cassidyjames**
Fixes #1183

### CODE REVIEW

**Comment by danrabbit** [comment]
I guess reading this, for our purposes does it really make it clearer to call it an `origin_package` here instead of `origin` and `origins`?

This would make our code things like `foreach (var origin in package.origins)` etc which sounds more like what it does even though it's not technically perfectly true

File: src/Views/AppInfoView.vala [hunk 1]
```diff
@@ -630,16 +630,16 @@ namespace AppCenter.Views {
 
             Gtk.TreeIter iter;
             uint count = 0;
-            foreach (var version in package.versions) {
-                version_liststore.append (out iter);
-                version_liststore.set (iter, 0, version, 1, version.origin_description);
-                if (version == package) {
-                    version_combo.set_active_iter (iter);
+            foreach (var origin_package in package.origin_packages) {
```
Comment (danrabbit) [inline src/Views/AppInfoView.vala]: While we're here should this be
```suggestion
            foreach (unowned Package origin_package in package.origin_packages) {
```

**Comment by cassidyjames** [comment]
@danrabbit that's what I had originally but it was similarly confusing. I think the more verbose but more correct terminology is advantageous because you immediately know it's a package and has all the properties and methods that implies.

**Comment by danrabbit** [comment]
Okay yup good point

**Pull request merged**